src/pagination/Pagination.js

```diff
@@ -138,9 +140,9 @@ export default class Pagination extends PureComponent {
     }
 
     render () {
-        const { dotsLength, containerStyle, vertical, accessibilityLabel } = this.props;
+        const { dotsLength, containerStyle, vertical, accessibilityLabel,renderSingleDot } = this.props;
 
-        if (!dotsLength || dotsLength < 2) {
+        if (!dotsLength || dotsLength < (renderSingleDot?1:2)) {
```

Comment:
please add space between the condition `(renderSingleDot ? 1 : 2)`, to increase the readability.

Reply:
Done